docs/getting-started.md

```diff
@@ -0,0 +1,171 @@
+
+```bash
+kubectl apply -f config/samples/httpbin.yaml
+```
+
+Or,
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+    name: httpbin
+spec:
+    name: httpbin
+    use_keyless: true
+    protocol: http
+    active: true
+    proxy:
+        target_url: http://httpbin.org
+        listen_path: /httpbin
+        strip_listen_path: true
+EOF
+```
+
+Let's walk through the ApiDefinition that we created. We have an ApiDefinition called `httpbin`, as specified in `spec.name` 
+field, that listens `/httpbin` and proxies requests to http://httpbin.org, as specified under `spec.proxy` field. Now, any 
+requests coming to the `/httpbin` endpoint will be proxied to the target URL that we defined in `spec.proxy.target_url`, 
+which is http://httpbin.org in our example.
+
+To find out about available ApiDefinition objects in your cluster:
+```bash
+$ kubectl get tykapis
+NAME      DOMAIN   LISTENPATH   PROXY.TARGETURL      ENABLED
+httpbin            /httpbin     http://httpbin.org   true
+```
+
+We can see that our ApiDefinition has been created. Now let's verify that our API is working as expected.
+
+**NOTE**: The verification step may vary based on your environment, such as the type of your Tyk installation and Kubernetes cluster.
+- If you are using local Kubernetes cluster such as [KinD](https://kind.sigs.k8s.io/) and [Minikube](https://minikube.sigs.k8s.io/docs/start/), 
+you can do port-forwarding to access objects within the cluster.
+- If you are using Kubernetes clusters provided by cloud providers, you need to configure your cluster to make it accessible.
+  
+For the scope of this example, we are using a local Kubernetes cluster. Port-forwarding details can be found in the 
+official [Kubernetes documentation](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/).
+
+### Tyk CE
+
+<details><summary>Example port forwarding for Tyk CE</summary>
+<p>
+
+If you have installed Tyk CE in the `<TYK_CE_NAMESPACE>` namespace, you will have the following services:
+```bash
+kubectl get svc -n <TYK_CE_NAMESPACE> 
+NAME                              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+gateway-svc-tyk-ce-tyk-headless   ClusterIP   10.96.38.138    <none>        8080/TCP   22m
+redis                             ClusterIP   10.96.254.227   <none>        6379/TCP   22m
+```
+
+In order to access Tyk Gateway, you can use the following port-forwarding command:
+```bash
+kubectl port-forward service/gateway-svc-tyk-ce-tyk-headless -n <TYK_CE_NAMESPACE> 8080:8080
+```
+
+The Tyk Gateway is accessible from your local cluster's 8080 port (e.g., `localhost:8080`).
+
+</p>
+</details>
+
+Since Tyk CE does not come with the Dashboard, you can list APIs using [Tyk Gateway API](https://tyk.io/docs/tyk-gateway-api/).
+
+```bash
+$ curl -H "x-tyk-authorization: foo" localhost:8080/tyk/apis/
```

Comment:
We should mention that `Replace `foo` with the `secret` set in tyk configuration.`